src/utilstrencodings.cpp

```diff
@@ -506,3 +506,25 @@ bool ParseFixedPoint(const std::string &val, int decimals, int64_t *amount_out)
     return true;
 }
 
+/// Parse the decimal string into a little-endian byte vector.
+std::optional<std::vector<unsigned char>> ParseArbitraryInt(const std::string& num_string)
```

Comment:
Is `unsigned char` different to, or more or less specific than `uint8_t`? I'm never clear on when one should choose one or the other.